packages/smooth_app/lib/pages/preferences/user_preferences_connect.dart

```diff
@@ -119,16 +123,60 @@ class UserPreferencesConnect extends AbstractUserPreferences {
                 // No email client installed on the device
                 showDialog<void>(
                   context: context,
-                  builder: (_) => SmoothAlertDialog(
-                    title:
-                        appLocalizations.no_email_client_available_dialog_title,
-                    body: Text(appLocalizations
-                        .no_email_client_available_dialog_content),
-                    positiveAction: SmoothActionButton(
-                      onPressed: () {
-                        Navigator.of(context, rootNavigator: true).pop();
-                      },
-                      text: appLocalizations.okay,
+                  builder: (BuildContext context) => ScaffoldMessenger(
+                    child: Builder(
+                      builder: (BuildContext context) => Scaffold(
```

Comment:
Why did you put it inside a scaffold, doesn't it work without that. I would think that it just uses the scaffold higher up in the widget tree.

Doesn't it work without that

Reply:
The snack bar was coming below the dialog, that's why I had to use a scaffold. Figured out from this [blog](https://medium.com/@aakashpp/flutter-snack-bar-on-dialog-ea70a3b5e028).

Reply:
Ohh okay that makes sense, could you then add a comment explaining this

Reply:
Yeah will do that !!